src/index.ts

```diff
@@ -169,6 +169,7 @@ export type { OrderedBulkOperation } from './bulk/ordered';
 export type { UnorderedBulkOperation } from './bulk/unordered';
 export type {
   ChangeStream,
+  ChangeStreamAggregateRawResult,
```

Comment:
It's not public it has to be exported to make it accessible from the top level import even though it's marked internal. I tried not exporting it but api-extractor still wants it exported probably has to do with how it's used as a generic on an imported function rather than entirely internal to the file like the TopologyOptions interface.